lib/student.rb

```diff
@@ -1,8 +1,12 @@
 require_relative 'person'
+
 class Student < Person
-  def initialize(classroom, age, name = 'Unknown')
+  attr_accessor :name, :classroom
+
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
     super(name, age, parent_permission)
     @classroom = classroom
+    classroom.students << self
   end
 
```

Comment:
- [ ] There should be a method that serves as the 'belongs-to' side (`a student belongs to a classroom`), something like so: 

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Please kindly fix that.